src/framerate.rs

```diff
@@ -0,0 +1,265 @@
+use num::{FromPrimitive, ToPrimitive};
+use std::fmt;
+use std::fmt::Formatter;
+use std::str::FromStr;
+
+/// NTSC is the type of NTSC standard a framerate adheres to.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum Ntsc {
+    /// False means this Framerate is not NTSC.
+    False(),
+    /// NonDrop means this Framerate is non-drop NTSC (no frame numbers are dropped to sync timecode
+    /// with real-world time - results in Timecode that drifts from true time).
+    NonDropFrame(),
+    /// DropFrame means this framerate is drop-frame NTSC (frames numbers are dropped periodically
+    /// to keep timecode in sync with real-world time).
+    DropFrame(),
+}
+
+impl Ntsc {
+    /// check returns whether this is any NTSC format (drop or non-drop).
+    pub fn check(self) -> bool {
```

Comment:
I think maybe I'll just keep an is_ntsc() which will be true if it's not the False() value, and get rid of the other two. 

It feels like there should just be an easy way to check for a positive value rather than checking if it's not not an NTSC value, which feels a little less readable.